doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
Honestly I don't think it belongs in multilocale.rst OR launchers.rst. I moved most of it from multilocale.rst because I wanted the instructions there to be shorter and more overview. I'd like multilocale.rst to be a jumping-off point, depending on configuration. I really don't want it to get bogged down in the details of UDP conduit since the UDP conduit isn't even supposed to be the "main" configuration.

Put another way, I'd rather have info about Cray XC and Infiniband in multilocale.rst than info about UDP. But those things clearly don't belong there.

If we moved the inforamtion about the UDP conduit to another file, where would we put it? As a sibling to infiniband.rst in platforms/ ? Or something else?